packages/node/src/login/oidc/refresh/TokenRefresher.ts

```diff
@@ -36,11 +36,17 @@ import { Issuer, TokenSet } from "openid-client";
 import { JWK } from "jose";
 import { configToIssuerMetadata } from "../IssuerConfigFetcher";
 
+// Some identifiers are in camelcase on purpose.
```

Comment:
I think this suggested change is more correct...?
```suggestion
// Some identifiers are *not* in camelCase on purpose, as they are named using the official names from the OIDC/OAuth2 protocols.
```

Reply:
Oops ^^. Resolved in bbba942939219f8c7351bee9bfa17ed20ee74dfe